dev/Debug.php

```diff
@@ -41,6 +41,17 @@ class Debug {
 	private static $friendly_error_header = 'There has been an error';
 
 	/**
+	 * Set to true to enable friendly errors to set a http response code corresponding to the error.
+	 * If left false then error pages will be served as HTTP 200.
+	 *
+	 * Will be removed in 4.0, and fixed to on.
+	 *
+	 * @config
+	 * @var bool
+	 */
+	private static $friendly_error_httpcode = false;
```

Comment:
We could also setup silverstripe-installer to set it to true so new installations get true but existing sites get false (since people rarely update silverstripe-installer). Bit tricky though, and probably breaks SemVer